analysis_options.yaml

```diff
@@ -3,3 +3,12 @@
 # BSD-style license that can be found in the LICENSE file.
 
 include: package:lints/recommended.yaml
+
+analyzer:
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
+  exclude:
+    - example/**
```

Comment:
`examples` folder contains flutter examples (which is currently a placeholder). After you are done with this review, I'll add the actual code and remove this line.
And, for `third_party` we can't ensure that if it's linted using `package:lints` rule or not. Even though they contains code from `dart:io`.

Reply:
Instead of excluding it entirely, you can consider adding `// ignore_for_file: lint1, lint2...`.
So that we know which lints can and cannot he handled.

Reply:
That sounds good to me.